### PR TITLE
chore: EoL for v4 and v5

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

closes: #XXXX

### Commit Message / Changelog Entry

```text
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: docs/versioned_docs/version-v4.5.x/00-intro.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ slug: /
 sidebar_position: 0
 ---
 
+:::danger
+This version of ibc-go is not supported anymore. Please upgrade to the latest version.
+:::
```
Comment (crodriguezvega) [inline docs/versioned_docs/version-v4.5.x/00-intro.md:8]: I added this in the latest release of v4 and v5 to warn readers.

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -66,8 +66,8 @@ const config = {
               path: "v5.3.x",
               banner: "none",
             },
-            "v4.4.x": {
-              path: "v4.4.x",
+            "v4.5.x": {
```
Comment (crodriguezvega) [inline docs/docusaurus.config.js:69]: I renamed the docs for v4.4 to v4.5, so that we keep the latest version.

**Review by damiannolan — APPROVED** [review]
There is a cron workflow in `e2e-upgrade.yml` as far as I know, which runs an upgrade test from v4 -> v5. We could also remove this? 

Also happy to keep it around for another while

**Review by colin-axner — APPROVED** [review]

**Comment by crodriguezvega** [comment]
> There is a cron workflow in `e2e-upgrade.yml` as far as I know, which runs an upgrade test from v4 -> v5. We could also remove this?
> 
> Also happy to keep it around for another while

I thought of removing it as well, but then thought that maybe it's still worth keeping it a bit, in case teams use it to test upgrades. But I agree that sooner rather than later we should also remove it.

**Pull request merged**